### DISCUSSION TITLE
How to set up SRS/DKIM

### DISCUSSION THREAD

**Discussion by idevai** [huan/docker-simple-mail-forwarder#41] [Q&A]
When forwarding to a google account, I get answers like "Our system has detected an unusual rate of 421-4.7.0 unsolicited mail originating from your IP address.". AFAIK this is because SRS/DKIM is not set up? How did you solve this?

**Comment by eugeis** [comment]
Hello, I would also like to know how to setup DKIM
Thank you for you cool forwarder

**Comment by riptidewave93** [comment]
I had a similar issue, and only needed to setup a SPF record (if you are forwarding mail). You may also want to make sure the IP you are sending from isn't on any blacklists. https://mxtoolbox.com/blacklists.aspx

**Comment by xriser** [comment]
SPF is not enough so I have added SRS/DKIM
Updated alpine/postfix version
https://github.com/xriser/docker-simple-mail-forwarder

There are some issues with passing tests with startls, but in general, it is working for me.

**Comment by gety9** [comment]
@xriser @huan why separate fork? Why not do PR and keep it in same repo/project?

Imho DKIM is a must feature, right now forwarded emails go to spam if forwarded to gmail

**Comment by xriser** [comment]
Hey, @gety9 
The changes I did, not good enough for the project.
I believe DKIM and SRS should be optional. So you should have some switches to enable and disable it.
The same things with their config files. Need to generate them somehow.
I am using simple-mail-forwarder in Kubernetes, so I have mounted configs with configmaps.
Also, I have removed the test user because it is a security vulnerability, and some other tests removed as well.
So, It is complicated for me to make such options for all the usage options. 
Anyway, @huan can pick-up any changes I did.
